randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -516,7 +541,8 @@ Extra - asset_id: 14
   * Normal Door to Terminator Room/Door to Parlor
   > Door to Landing Site
       Any of the following:
-          Screw Attack or Speed Booster
+          Screw Attack
+          Speed Booster and Disabled Entrance Randomizer
```

Comment:
This would also need disabled door lock rando, as otherwise you might be unable to open the door.

Reply:
Added.